### PR TITLE
Edit a session from the timeline

### LINKED ISSUE #513: Edit a session from the timeline
Allow the organizer to edit a session from the Scheduler UI's timeline itself. The organizer must be able to modify basic aspects of the session (title, description, abstract timings, etc)

(As suggested by @rafalkowalski in #502 )

### LINKED ISSUE #516: Save the edit session info to backend[Schedule UI]
- [x] Submit form data through ajax. (#673)
- [x] Update info in the schedule UI frontend. (#673)

### PR DESCRIPTION
**Pull request by niranjan94**
Allow the organizer to edit a session from the Scheduler UI's timeline itself. The organizer must be able to modify basic aspects of the session (title, description, abstract timings, etc)

Resolves #513. Partially resolves #516. 

@SaptakS @rafalkowalski  please review and merge into `development`

### CODE REVIEW

**Comment by SaptakS** [comment]
Seems good to merge.
